tests/phpunit/Integration/JSONScript/TestCases/scite-14-fixed-property-query.json

```diff
@@ -19,6 +19,9 @@
 		{
 			"about": "#0 non display formatted query output",
 			"subject": "Example/Query/1",
+			"skip-on": {
+				"mediawiki": [ ">1.38.x", "Check for MW 1.39+" ]
```

Comment:
Does not seem like a good message/description. "Skip on MW 1.39 and above" is a lot clearer, and you could also add something about why the skip is in place.

Reply:
Thanks, done.